src/storage/storages.rs

```diff
@@ -174,15 +175,13 @@ unsafe impl<T> DistinctStorage for NullStorage<T> {}
 pub struct VecStorage<T>(Vec<T>);
 
 impl<T> UnprotectedStorage<T> for VecStorage<T> {
-    unsafe fn clean<F>(&mut self, has: F)
```

Comment:
Did you benchmark this?

Reply:
Nope.
I'll try to compare old way (but checking index directly in bitset instead of function)
and iteration over bitset.

Reply:
Did you do this?

Reply:
There was no significant difference as expected.